src/Fable.Fetch.fsproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Fable.Browser.Blob" Version="1.1.0" />
-    <PackageReference Include="Fable.Core" Version="3.1.5" />
+    <PackageReference Include="Fable.Core" Version="3.2.3" />
```

Comment:
Because this is a library, it's better not to request the latest Fable.Core version unless it's really necessary. This way we avoid the need for consumers to update Fable.Core just in order to update Fable.Fetch.